scrapers/Karissa-Diamond.yml

```diff
@@ -0,0 +1,32 @@
+name: 'Karissa Diamond'
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - karissa-diamond.com/videoCollection
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@id="setTitle"]/text()
+      Studio:
+        Name:
+          fixed: "Karissa Diamond"
+      Date:
+        selector: //div[@id="setDate"]/text()
+        postProcess:
+          - parseDate: January 02, 2006
+      Details: //div[@class="well"]/text()
+      Tags:
+        Name: //div[@class="col-md-5 col-md-offset-1 col-md-push-6 hidden-xs"]//div[@class="col-xs-12 tagList"]/a/text()
+      Performers:
+        Name:
+          fixed: "Karissa Diamond"
+      Image:
+        selector: //meta[@property="og:image"]/@content
+        postProcess:
```

Comment:
I have to wonder if this is better to have commented out as an option for the user to turn on but not on by default.